sites/www.zooniverse.org.conf

```diff
@@ -86,11 +86,31 @@ server {
 
         resolver 8.8.8.8;
         if ($request_method ~ ^(GET|HEAD)$) {
-           proxy_pass             http://zooniverse-static.s3-website-us-east-1.amazonaws.com/$host/;
+            proxy_pass             http://zooniverse-static.s3-website-us-east-1.amazonaws.com/$host/;
+            set $proxy_host_header "zooniverse-static.s3-website-us-east-1.amazonaws.com";
         }
         if ($request_method = POST) {
-           proxy_pass             https://panoptes.zooniverse.org$uri;
+            proxy_pass             https://panoptes.zooniverse.org$uri;
+            set $proxy_host_header "panoptes.zooniverse.org";
         }
-        include /etc/nginx/proxy-headers.conf;
+        add_header 'Access-Control-Allow-Origin' '*';
```

Comment:
these cors requests should not be set here, they should be set by the upstream proxy.

these cors headers effectively add any domain is allowed to post requests, we control this at a finer level at the api, https://github.com/zooniverse/panoptes/blob/d0eadcf11d61f4510d92cf73f0c1411104de3ce1/config/initializers/cors.rb#L10-L13